bbc/simorgh
# WSTEAM1-1271 - Change `av-embeds` AMP url

Part of [WSTEAM1-1271](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1271)

Overall changes
======
- Changes AMP iframe src URL to use newer domain. This should behave exactly the same as the current domain, just under a different alias

Code changes
======
- Updates references to point to newer domain for AMP media iframes
- Updates tests / snapshots

Testing
======
1. Visit http://localhost:7080/mundo/articles/c1xv2q1gewvo.amp?renderer_env=test
2. Confirm the media plays back correctly
3. Inspect the iframe url and confirm it points to the new domain
4. Visit http://localhost:7080/serbian/lat/23279016.amp?renderer_env=test
5. Confirm the media plays back correctly
6. Inspect the iframe url and confirm it points to the new domain

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
